package.json

```diff
@@ -33,6 +33,7 @@
         "SFCA"
     ],
     "dependencies": {
+        "@salesforce/sfdx-scanner": "3.22.0",
```

Comment:
If we do end up shipping this... then we'll want to swap this out for 4.0.0 immediately after we release the 4.0.0 cli.

Reply:
Updated this to 4.0.0.